src/annotator/sidebar.js

```diff
@@ -57,15 +59,18 @@ export default class Sidebar {
    * @param {HTMLElement} element
    * @param {import('./util/emitter').EventBus} eventBus -
    *   Enables communication between components sharing the same eventBus
-   * @param {Guest} guest -
+   * @param {Pick<Guest, DependenciesSidebar|DependenciesBucketBar>} guest -
    *   The `Guest` instance for the current frame. It is currently assumed that
    *   it is always possible to annotate in the frame where the sidebar is
    *   displayed.
    * @param {Record<string, any>} [config]
    */
   constructor(element, eventBus, guest, config = {}) {
     this._emitter = eventBus.createEmitter();
-    const hypothesisAppsOrigin = new URL(config.sidebarAppUrl).origin;
+    this._guest = guest;
+    this._config = config;
```

Comment:
Rather than saving the whole config object as a field we could just save the data that other methods actually use, which looks like just `_showHighlights`.